### PR TITLE
Fixed #67 old properties are now deprecated

### LINKED ISSUE #67: Some properties for sunstone do not start with "sunstone."
These two:
`clouds.sysprop.value.delimiter` at `ObjectProperties`
`clouds.ssh` at `DynamicSshClientModule`

They should be deprecated and replaced with properties with proper prefix.

And the Arquillian README mentions old properties with the "clouds" prefix.

### PR DESCRIPTION
**Pull request by LittleJohnII**
@mjurc fixing a few issues related to properties, this is very loosely related to the "Variable number of nodes with the same configuration" task for the clustering test suite.

@simkam and @kwart may also be interested.

### CODE REVIEW

File: core/src/main/java/org/wildfly/extras/sunstone/api/impl/ObjectProperties.java [hunk 1]
```diff
@@ -26,12 +26,19 @@
...
 public class ObjectProperties implements ConfigProperties {
     private static final Logger LOGGER = SunstoneCoreLogger.DEFAULT;
 
     public static final String PROPERTIES_FILE_KEY = "properties";
+    /**
+     * Replaced by {@link ObjectProperties#SYSTEM_PROPERTY_VALUE_DELIMITER_SST}.
+     */
+    @Deprecated
     public static final String SYSTEM_PROPERTY_VALUE_DELIMITER = "clouds.sysprop.value.delimiter";
+    public static final String SYSTEM_PROPERTY_VALUE_DELIMITER_SST = "sunstone.sysprop.value.delimiter";
```
Comment (mjurc) [inline core/src/main/java/org/wildfly/extras/sunstone/api/impl/ObjectProperties.java]: What is the _SST here? Just for clarification for me.

Comment (LittleJohnII) [reply core/src/main/java/org/wildfly/extras/sunstone/api/impl/ObjectProperties.java]: That's a short name for Sunstone.

Comment (mjurc) [reply core/src/main/java/org/wildfly/extras/sunstone/api/impl/ObjectProperties.java]: If you're fine with abbreviation, I'm fine too, I just don't see the point in abbreviating for already long name. :)

**Comment by LittleJohnII** [comment]
Thanks for the reviews, merging.

**Pull request merged**